sdk/template/Azure.Template/src/Generated/MultiVersionClientOptions.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.Template
+{
+    /// <summary> Client options for MultiVersionClient. </summary>
+    public partial class MultiVersionClientOptions : ClientOptions
+    {
+        private const ServiceVersion LatestVersion = ServiceVersion.V2022_02_02;
+
+        /// <summary> The version of the service to use. </summary>
+        public enum ServiceVersion
+        {
+            /// <summary> Service version "2022-01-01". </summary>
+            V2022_01_01 = 1,
+            /// <summary> Service version "2022-02-02". </summary>
+            V2022_02_02 = 2,
+        }
+
+        internal ServiceVersion TargetVersion { get; }
```

Comment:
internal Version stored as enum to enable using comparison operators to test whether an API is supported by target service version.